EthOn.rdf

```diff
@@ -774,7 +775,7 @@
     <owl:DatatypeProperty rdf:about="http://ethon.consensys.net/accountCodeHash">
         <rdfs:subPropertyOf rdf:resource="http://ethon.consensys.net/AccountDataProperty"/>
         <rdf:type rdf:resource="http://www.w3.org/2002/07/owl#FunctionalProperty"/>
-        <rdfs:domain rdf:resource="http://ethon.consensys.net/ContractAccount"/>
+        <rdfs:domain rdf:resource="http://ethon.consensys.net/Account"/>
```

Comment:
On third thought, even though this is not what is stated in the PR comment, it is reasonable and can be merged.